.rubocop_todo.yml

```diff
@@ -89,6 +89,7 @@ RSpec/VerifiedDoubles:
 
 RSpec/AnyInstance:
   Exclude:
+    - 'spec/controllers/api/generic_works_controller_spec.rb'
```

Comment:
You can allow `allow(MyClass).to receive(:find).and_return(double)` or `allow(MyClass).to receive(:new).and_return(double)` and then call your expectation on that double.. You could do that with all the grandfathered exceptions if you wanted to.